### DISCUSSION TITLE
'Captcha:' shows in form

### DISCUSSION THREAD

**Discussion by timfidd** [django-recaptcha/django-recaptcha#259] [Q&A] [answered]
I there any way to not display 'Captcha:' in a form ?

**Comment by MonsignorEduardo** [comment]
being captcha = ReCaptchaField(...)
 captcha.label=False

**Comment by marksweb** [comment]
```Python
    captcha = ReCaptchaField(
        label='',
        widget=ReCaptchaV3()
    )
```

**Comment by cpattersonit** [comment, marked as answer]
Another (more accessible) way to do this would be to use JavaScript to insert a class that keeps the label for screen readers but hides it otherwise::

    $("label[for=id_captcha]").addClass("sr-only");

This example relies on jQuery and Bootstrap, but you could roll your own solution to accomplish the same thing.
